sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-verifyAssessmentCalculationOrder.e2e-spec.ts

```diff
@@ -536,6 +538,223 @@ describe("AssessmentController(e2e)-verifyAssessmentCalculationOrder", () => {
     });
   });
 
+  it("Should sum the grants from past applications with different offering intensities when the applications are for the same student and program year.", async () => {
```

Comment:
It is my understanding that we already have an E2E for grants. This E2E can be focused on the workflow totals instead.